recipe/meta.yaml

```diff
@@ -13,6 +13,7 @@ source:
 
 build:
   number: 0
+  skip: True  # [py>35]
```

Comment:
Looks good to me. We can also work on getting the dependencies compiled for `Python 3.6` and make it work.